charts/nginz/templates/conf/_nginx.conf.tpl

```diff
@@ -186,7 +186,7 @@ http {
   #
 
   server {
-    listen {{ .Values.config.http.httpPort }};
+    listen {{ .Values.config.http.httpPort }}{{ if (.Values.config.http2.enabled) }} http2 {{end}};
```

Comment:
Why should offering to connect via http2 be a problem? I'd guess it's more about clients, and making sure their http2 implementations are not broken.

Reply:
the ingress controller is in front of nginz; and I'm not sure if it talks http2 to upstreams or not. Hence this PR to try out what happens.